.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -13,6 +13,12 @@ Before proceeding, please update to latest version and test if the issue persist
 ## Describe the bug in a sentence or two.
 …
 
+## Describe the desired/expected behavior.
+…
+
```

Comment:
I would also add a section for reproduction steps and a section for versions used in core and react:)

Reply:
already exists, but updated it, please see last commit